### DISCUSSION TITLE
Tuple Concatenation Error in create_group Method on PyAEDT 0.9.11

### DISCUSSION THREAD

**Discussion by mchoi327** [ansys/pyaedt#5058] [Q&A] [answered]
**Description:**

@lorenzovecchietti , @Samuelopez-ansys , In PyAEDT version 0.9.11 (Icepak), I encountered an issue when trying to group multiple objects using modeler.create_group. The function works correctly in version 0.9.9, but after updating to 0.9.11, it throws an error related to concatenating lists and tuples.

**Steps to Reproduce:**

Use PyAEDT 0.9.11.
Create multiple objects (in my case, PS boxes) and attempt to group them using modeler.create_group(objects=ps_boxes, group_name=ps_group_name).
The error occurs when trying to group a list of objects into a group.

**Example code that triggers the error**
ps_group_name = f"PS_Group_2"
ps_group = ipk.modeler.create_group(objects=ps_boxes, group_name=ps_group_name)

**Error Log:**
PyAEDT ERROR: Method arguments: 
PyAEDT ERROR:     objects = ['PS_0_2', 'PS_0_2_8', 'PS_0_2_9', ... ] 
PyAEDT ERROR:     group_name = PS_Group_2 
PyAEDT ERROR: **************************************************************
PS boxes grouped as: False
PyAEDT ERROR: **************************************************************
...
PyAEDT ERROR:   File "c:\Users\Mingeun\Anaconda3\envs\ansys_test\Lib\site-packages\pyaedt\modeler\cad\Primitives.py", line 5607, in create_group
PyAEDT ERROR:     Can only concatenate list (not "tuple") to list on create_group

**Environment:**

PyAEDT Version: 0.9.11**
Python Version: 3.12.3
_This code worked without issues in PyAEDT version 0.9.9. The problem seems to have arisen after upgrading to 0.9.11._

**Expected Behavior:**

The create_group method should successfully create a group without raising a tuple concatenation error.

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @mchoi327 ,

I am using Python 3.10 and Pyaedt v0.10.0, and it works for me:

```
from ansys.aedt.core import Icepak

aedtapp = Icepak(version="2024.2")

solids = ["Box1", "Box2"]

ps_group_name = f"PS_Group_2"

ps_group = aedtapp.modeler.create_group(objects=solids, group_name=ps_group_name)


aedtapp.release_desktop(False, False)
```

**Reply by mchoi327** [reply]
Thanks. I confirmed with PyAEDT v0.10.0.
